src/attributes/classes.js

```diff
@@ -7,16 +7,20 @@ define([
 
 var rclass = /[\t\r\n\f]/g;
 
+function getClass( elem ) {
+	return elem.getAttribute && elem.getAttribute( "class" ) || "";
```

Comment:
> getAttribute can return null and this ensures we return a string.

Aaam, if you said something wrong, i didn't notice :-), i meant it is good if some function always returns the same type.
I wondering about what tests we wouldn't pass though, if it would be just `elem.getAttribute( "class" ) || ""`?